### PR TITLE
Make layer compatible with linux 4.4

### PR DESCRIPTION
**Pull request by jeanparpaillon**

### CODE REVIEW

**Comment by mickey-happygolucky** [comment]
Hi 
Thank you for your cooperation.

Btw, recipes-kernel/linux/linux-raspberrypi/support_rpi_pitft35_44.patch~ seems backup file.
This is need?

**Comment by jeanparpaillon** [comment]
Right, I've just updated the pr

File: recipes-kernel/linux/linux-raspberrypi_4.4.bbappend [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += "file://support_rpi_pitft35.patch \
```
Comment (mickey-happygolucky) [inline recipes-kernel/linux/linux-raspberrypi_4.4.bbappend:3]: I think, support_rpi_pitft35.patch should be support_rpi_pitft35_44.patch.
Is this what you intended?

**Comment by mickey-happygolucky** [comment]
Hi, Jean

I checked. however could not build successfully.
Can you fix following two points?
- support_rpi_pitft35.patch -> support_rpi_pitft35_44.patch in linux-raspberrypi_4.4.bbappend
- rename from linux-raspberrypi_%.bbappend to linux-raspberrypi_3.18.bbappend

**Comment by jeanparpaillon** [comment]
Hi, I've finally pushed the changes to meta-raspberrypi layer 
See https://github.com/agherzan/meta-raspberrypi/pull/24

**Pull request closed without merging**